README.md

```diff
@@ -58,13 +58,13 @@ expo: {
 }
 ```
 
-## Autolinking
-
-This library is correctly autolinked on React Native. It can be used in both React Native and Expo projects.
-
 ## Usage
 
-```javascript
+This library can be used in both React Native and Expo projects. Note however:
```

Comment:
There were some questions on how to use this native module, this should explain things better.